codes/classical/properties/dual.yml

```diff
@@ -22,6 +22,9 @@ description: |
 protection: 'The dual of an \([n,k,d] \) code
   is an \([n,n-k,d'']\) code, where \(d''\) is not always related to \(d\). The generator matrix of \(C^\perp\) is the parity check matrix of \(C\), and visa versa.'
 
+notes:
+  - 'See \cite{arXiv:1706.01694}, which contains classing information concerning variations of extremal self-dual codes.'
```

Comment:
I think you hit a rabbit hole. Check the Preface of the book "Self-dual codes and invariant theory", and try to add the most relevant definitions and papers from there (e.g., Gleason's theorem, https://ieeexplore.ieee.org/document/59931, etc). May be more appropriate to add some of the them to code_id: divisible or code_id: golay instead of here.

Re the above:
'See \cite{...} for constructions of binary self-dual codes of length 60.'